UnityPy/helpers/MeshHelper.py

```diff
@@ -650,41 +654,31 @@ def get_triangles(self) -> List[List[Tuple[int, ...]]]:
             indexCount = m_SubMesh.indexCount
             topology = m_SubMesh.topology
 
-            triangles: List[Tuple[int, ...]]
+            triangles: List[Tuple[int, ...]] = []
```

Comment:
There is no point is already assigning a value here, as we don't know the size yet.
Initializing the list with the correct size and writing into it is always faster than appending, as the later requires at least one, usually multiple reallocs, which are expensive.